src/si/torque_magnitude.rs

```diff
@@ -0,0 +1,155 @@
+//! Torque (aka moment of force) (base unit newton meter, kg<sup>1</sup> · m<sup>2</sup> ·
+//! s<sup>-2</sup>).
+
+quantity! {
+    /// Torque magnitude (base unit newton meter, kg<sup>1</sup> · m<sup>2</sup> · s<sup>-2</sup>).
+    ///
+    /// Torques are moments, by which I mean they inherently depend on a distance to a frame of
```

Comment:
Use implicit second person for comments (https://developers.google.com/style/person). I think that is the correct description. Don't use "I".